README.md

```diff
@@ -35,7 +35,7 @@ Biopython is an optional dependency and not installing it will simply result in
 
 Run
 
-   python -m Orange.canvas
+	python -m Orange.canvas
```

Comment:
This is a [tab character](https://raw.githubusercontent.com/ajdapretnar/orange3-text/523ae01d10d9751245f4f333c276d01b46d7e09d/README.md), should be four spaces.